textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexInfo.java

```diff
@@ -0,0 +1,79 @@
+package edu.uci.ics.textdb.dataflow.regexmatch;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @Author Zuozhi Wang
+ * @Author Shuying Lai
+ * 
+ * RegexInfo for translating regex to trigrams
+ * @see https://swtch.com/~rsc/regexp/regexp4.html for details
+ */
+public class RegexInfo {
+	boolean emptyable;
+	List<String> exact = null;
+	List<String> prefix = null;
+	List<String> suffix = null;
+	TrigramBooleanQuery match = null;
```

Comment:
I think I looked at an old version.  Still it's good to run the `grep` command to find the remaining ones.